ttnn/cpp/ttnn/operations/eltwise/binary/device/binary_composite_op.cpp

```diff
@@ -264,6 +264,19 @@ Tensor _div_no_nan(const Tensor& input_a, const Tensor& input_b, const std::opti
     return ttnn::where(ttnn::eqz(input_b, output_mem_config), 0, div_result);
 }
 
+Tensor ExecutePrelu::invoke(const Tensor& input, float scalar, const std::optional<MemoryConfig>& output_mem_config) {
+    return ttnn::prelu_sfpu(input, scalar);
+}
+
+Tensor ExecutePrelu::invoke(const Tensor& input_a, const Tensor& input_b, const std::optional<MemoryConfig>& output_mem_config) {
+    const tt::tt_metal::LegacyShape s_a = input_a.get_legacy_shape();
+    auto volume = input_b.get_logical_volume();
+    // If volume = 1 Support for a single-value tensor yet to be handled. TODO(#14933)
+    TT_FATAL(s_a[1] == volume, "Mismatch of parameter numbers and input channel size");
```

Comment:
can we use s_a[-3] ? 
s_a[1] will refer to channel dim only when input tensor is of rank 4.
and also add TT_FATAL to check whether input tensor is of rank >= 3 since this op works on channel dim